### DISCUSSION TITLE
Yi Home 1080p AI+ Support - SFUS, 12.2.0.2*

### DISCUSSION THREAD

**Discussion by Pragtically** [roleoroleo/yi-hack-Allwinner-v2#725] [Q&A] [answered]
Can't seem to find this specific Serial/Firmware combination. Is this supported/in development? Thanks!

**Comment by roleoroleo** [comment]
Unsupported at the moment.
Please share a backup of the flash.

**Reply by Pragtically** [reply]
Hope this helps the effort. Thanks for all your work!

LINK REMOVED

**Comment by roleoroleo** [comment, marked as answer]
You cam is a y20ga
Check this repo:
https://github.com/roleoroleo/yi-hack-Allwinner

**Reply by Pragtically** [reply]
Confirming the release for y20ga in the Allwinner-v1 repo worked for SFUS serial and 12.2.0.2 firmware.
Could you please elaborate what information from the backup files allowed you to know the cam model? Thank you again.

**Reply by roleoroleo** [reply]
mtdblock3 contains a jffs2 filesystem.
If you unpack it, there are a lot of files where you can find the model.
For example /home/base/tools/extpkg.sh
